### DISCUSSION TITLE
gesture to switch workspaces with touchscreen?

### DISCUSSION THREAD

**Discussion by sound-friend** [paperwm/PaperWM#918] [Q&A]
hey! this may be a silly question, but i love using this on touchscreens! however, navigating between workspaces is a pain with touch only. is there a gesture to let me switch between workspaces (similar to super+ pageup/down) that i'm missing? thanks for the help!

**Comment by jtaala** [comment]
Not a silly question.

@Thesola10 has implement most touchscreen functionality in PaperWM (I don't have a touchscreen and hence haven't been much help here).

I don't believe there is currently a gesture for workspace switching for touch (like there is for touchpad).  @Thesola10 can confirm though.

**Reply by Thesola10** [reply]
IIRC GNOME has a builtin gesture to switch workspaces, which is a three finger swipe on the touchscreen.

Personally I use [Overview Flick](https://extensions.gnome.org/extension/6478/overview-flick/) combined with [V-Shell](https://extensions.gnome.org/extension/5177/vertical-workspaces/) so the workspaces are directly under my right thumb when I open Overview
